src/jit/gtlist.h

```diff
@@ -226,6 +226,12 @@ GTNODE(JCC              , GenTreeCC          ,0,GTK_LEAF|GTK_NOVALUE)   // Check
 GTNODE(SETCC            , GenTreeCC          ,0,GTK_LEAF)               // Checks the condition flags and produces 1 if the condition specified 
                                                                         // by GenTreeCC::gtCondition is true and 0 otherwise.
 
+#ifdef _TARGET_XARCH_
+GTNODE(BT               , GenTreeOp          ,0,GTK_BINOP|GTK_NOVALUE)
+GTNODE(BTC              , GenTreeOp          ,0,GTK_BINOP)
```

Comment:
I would prefer that we not add the additional forms as GenTree operators until they are going to be used.

Reply:
Also, it would be good to add comments describing them.

Reply:
> I would prefer that we not add the additional forms as GenTree operators until they are going to be used.

Will do, they're easy to add back if needed. And it may turn out that they're never needed because they have worse perf characteristics on AMD CPUs. I'm not sure it's worth the trouble to make the JIT use them only on Intel CPUs.